commands/resource_commands/deploy

```diff
@@ -582,7 +583,7 @@ if [[ $DB_TYPE == "$INTERNAL_DB" ]]; then
         check_and_install_kbcli
     fi
 
-    if ! kbcli cluster list 2>/dev/null || ! kbcli kubeblocks status; then
+    if kbcli cluster list | grep "No cluster found"; then
```

Comment:
this is the correct way to check is kubeblocks cluster is present or not right?

Reply:
@grapple-solutions

Reply:
@zahid-iqbal-alpha 
I think you need to have 3 steps:
- first, check if the cli ist installed - if not, install the cli:
if ! kbcli >/dev/null 2>&1; then
- second, check if kubeblocks is installed in the kubernetes cluster:
if ! k get clusters.apps.kubeblocks.io >/dev/null 2>&1; then
- third, check if the db cluster is already created:
if kbcli cluster list -n ${KUBE_DB} ${GRAS_NAME} >/dev/null 2>&1; then

I think then it would be perfect...
Thanks

images:
<img width="1253" alt="image" src="https://github.com/grapple-solutions/grapple-cli/assets/130764623/69963ec9-a924-4033-944a-2bb8ea6ce4e1">
<img width="1139" alt="image" src="https://github.com/grapple-solutions/grapple-cli/assets/130764623/82bb2b9e-4e43-4019-b37e-274440aef6c7">